pro/src/new_components/StockEventForm/StockEventForm.stories.tsx

```diff
@@ -0,0 +1,105 @@
+// eslint-disable-next-line import/named
+import { ComponentStory } from '@storybook/react'
+import { Formik } from 'formik'
+import React from 'react'
+
+import FormLayout from 'new_components/FormLayout'
+import { SubmitButton } from 'ui-kit'
+import { getToday } from 'utils/date'
+import { getLocalDepartementDateTimeFromUtc } from 'utils/timezone'
+
+import {
+  StockEventForm,
+  STOCK_EVENT_FORM_DEFAULT_VALUES,
+  getValidationSchema,
+  IStockEventFormValues,
+} from '.'
+
+export default {
+  title: 'components/StockEventForm',
```

Comment:
je me dis qu'on pourrait rassembler toutes les stories sur les stocks dans le même dossier pour plus de clarté
```suggestion
  title: 'components/stocks/StockEventForm',
```

et faire la même modif dans les autres stories